streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -424,7 +424,7 @@ public void shouldUseSpecifiedNameForStreamSourceProcessor() {
         builder.stream(STREAM_TOPIC_TWO);
         builder.build();
         final ProcessorTopology topology = builder.internalTopologyBuilder.rewriteTopology(new StreamsConfig(props)).build();
-        assertSpecifiedNameForOperation(topology, expected, "KSTREAM-SOURCE-0000000000");
+        assertSpecifiedNameForOperation(topology, expected, "KSTREAM-SOURCE-0000000001");
```

Comment:
First source node would be `KSTREAM-SOURCE-0000000000` so this is a valid change in the test